docs/source/faq.rst

```diff
@@ -39,6 +39,52 @@ As a consequence, the following code is correct too:
 More details `Events and Handlers <concepts.html#events-and-handlers>`_.
 
 
+Creating Custom Events based on Forward/Backward Pass
+-----------------------------------------------------
+
+There are cases where the user might want to add events based on the loss calculation and backward pass. Ignite provides
+flexibility to the user to allow for this:
+
+.. code-block:: python
+
+    class BackpropogationEvents(Enum):
```

Comment:
BackpropAgationEvents and same for others

Reply:
To clarify are you asking to edit the name?

Reply:
typo: BackpropogationEvents ->BackpropAgationEvents